docs/app-directory/overview.md

```diff
@@ -1,24 +1,171 @@
+
+AppD reduces fragmentation in the market and allows end-users more flexibility in what applications can be included in their desktop.
+
+#### Intent Resolution
+
+AppD provides information to the Desktop Agent on which applications can handle particular [intents](../intents/overview) and the context types they support as input to them. This allows the Desktop Agent to implement an [intent resolver](../api/spec#resolvers) that can launch applications and pass the intent and context to them to operate on, supporting workflows between applications that didn't require prior bilateral agreements between the application providers.
+
+### For an Application Provider
+
+Until now, we've looked at appD from the perspective of a desktop owner and user. But appD also offers advantages to vendors who develop apps for the financial services desktop.
+
+#### Apps Work Well Together Out-of-the-box
+
+When your customers add your FDC3-compliant app to their desktop via an appD record accurately describing it, you can be sure that your app will interoperate with other apps that follow the FDC3 standard. You don’t have to do anything special, or arrange a bilateral agreement with anyone else. The benefit of the open standard is that any app that follows it will work well with any other compliant app.
+
+#### Easy Updates
+
+As a vendor, you prefer for all your customers to run your latest software. However, many customers will postpone upgrades, sometimes for a long time, because upgrading can be a pain. An advantage of a vendor-hosted appD is that the configuration of an app can be updated at any time and, if your customers need to choose when to upgrade, multiple versions of it can be made available, each with their own configuration. By making it easier for customers to update, you can drive better adoption.
+
+### For the Industry
+
+By hosting our own appD we can easily combine applications from various providers into one cohesive directory. Alternatively, we can connect to directories from multiple providers (in standardized format) and provide a single view over them. This reduces fragmentation in the market, allows end-users more flexibility in what apps to include in their smart desktop, and obviates the need for vendors to provide application details in diverse formats or for their customers to work out these details for themselves.
+
+## Use Cases
+
+An application directory (AppD) provides information about an application's identifiers, publisher details, intents that it supports, and metadata necessary to launch and integrate the application in a desktop agent.
```

Comment:
Until this point on the page the 'a' in 'appD' has been lower case. I don't really have a preference but we should be consistent.

Reply:
it seems to be all 'AppD' after this

Reply:
Duly noted - product of merging different docs. We should always use whatever capitalisation is defined in the glossary (now that we have one), where it is lowercase.
```suggestion
An application directory provides information about an application's identifiers, publisher details, intents that it supports, and metadata necessary to launch and integrate the application in a desktop agent.
```